Cargo.toml

```diff
@@ -22,20 +22,22 @@ serde_json = "1.0"
 thiserror = "1.0"
 url = "2.2"
 image = "0.23"
+mime_guess = "2.0.3"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 cairo-rs = "0.9"
 webkit2gtk = { version = "0.11", features = ["v2_8"] }
 gio = "0.9"
+glib = "0.10.3"
```

Comment:
Should I undo that move or does it not matter?

Reply:
no need, "0.10.3" actually means "0.10.3+" so I think it's the same effect to "0.10"